cfglib.py

```diff
@@ -172,6 +172,8 @@
                 help="Don't show ssl warnings"),
     cfg.BoolOpt('keep_affinity_settings', default=False,
                 help="Keep affinity/anti-affinity settings"),
+    cfg.StrOpt('ssh_cipher', default='arcfour128',
```

Comment:
The problem is that arcfour may not be supported by all SSH servers, here are few examples I was able to find:
 - http://labs.hoffmanlabs.com/node/1897
> Per the OpenSSH 6.7 release notes, "The default set of ciphers and MACs has been altered to remove unsafe algorithms. In particular, CBC ciphers and arcfour* are disabled by default."

 - http://mgalgs.github.io/2014/10/22/enable-arcfour-and-other-fast-ciphers-on-recent-versions-of-openssh.html
> The no matching cipher found error message is a result of OpenSSH 6.7 disabling a few ciphers by default for security reasons.

So it would be much better to **not** use arcfour as default